clusterloader2/pkg/tuningset/randimized_load.go

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
s/randomized/randomized/ in file name :)

Reply:
Done.